source/extensions/omni.isaac.lab/test/sensors/test_frame_transformer.py

```diff
@@ -71,7 +81,7 @@ def setUp(self):
         # Create a new stage
         stage_utils.create_new_stage()
         # Load kit helper
-        self.sim = sim_utils.SimulationContext(sim_utils.SimulationCfg(dt=0.005))
+        self.sim = sim_utils.SimulationContext(sim_utils.SimulationCfg(dt=0.005, device="cpu"))
```

Comment:
do we only want to test cpu?

Reply:
For now, yes as it would require a bit more change to update all these test cases to do on CPU / GPU as `test_articulation.py` does